custom/aaa/dbaccessors.py

```diff
@@ -0,0 +1,306 @@
+from __future__ import absolute_import, unicode_literals
+
+from dateutil.relativedelta import relativedelta
+
+from custom.aaa.models import Child, ChildHistory
+
+
+class ChildQueryHelper(object):
+    def __init__(self, domain, person_case_id):
+        self.domain = domain
+        self.person_case_id = person_case_id
+
+    def infant_details(self):
+        extra_select = {
+            'breastfeedingInitiated': 'breastfed_within_first',
+            'dietDiversity': 'diet_diversity',
+            'birthWeight': 'birth_weight',
+            'dietQuantity': 'diet_quantity',
+            'breastFeeding': 'comp_feeding',
+            'handwash': 'hand_wash',
+            'exclusivelyBreastfed': 'is_exclusive_breastfeeding',
+            'babyCried': 'child_cried',
+            'pregnancyLength': "'N/A'",
+        }
+
+        return Child.objects.extra(
```

Comment:
it looks like using `extra` is deprecated: https://docs.djangoproject.com/en/2.1/ref/models/querysets/#django.db.models.query.QuerySet.extra
Are you just doing this because these columns are aliased? I think you might be able to use an `F()` expression instead. (https://docs.djangoproject.com/en/1.11/ref/models/expressions/#using-f-with-annotations)

Reply:
I was limiting this PR as much as possible to just moving the logic from models -> dbaccessors and creating tests. I have noted this and it will be followed up on (and end goal would to not use either extra or F, but to only return python style names and let JS handle the conversion): https://github.com/dimagi/commcare-hq/pull/23664#pullrequestreview-216883230